hibernate-core/src/test/java/org/hibernate/jpa/test/query/QueryTest.java

```diff
@@ -704,6 +704,138 @@ public void testJpaPositionalParameters() {
 			em.close();
 		}
 	}
+	
+	/**
+	 * Collection parameters are internally rewritten to named parameters (one named parameter by
+	 * collection item) and HQL parser rejects mixed positional and named parameters queries.
+	 */
+	@Test
+	@TestForIssue(jiraKey = "HHH-12290")
+	public void testParameterCollectionAndPositional() {
+		final Item item = new Item( "Mouse", "Micro$oft mouse" );
```

Comment:
Let's be neutral here and use `Microsoft`.

Reply:
@gsmet there are 33 occurrences of `Micro$oft` in this file. Should i change all of them?

Reply:
Argh OK, it's an existing pattern. Sigh. I really don't like this sort of things.

I would be for not adding more of them.

And maybe send another PR cleaning this up, once this one gets accepts (better not to add noise to this one).

Reply:
These have been here since we have moved from CVS which is what, 10 years now?  I think we have enough work to not fret over such silly things.  But by all means, if someone wants to take the time I won't stop them.

Reply:
Yeah, I just think it does not give a good image of the project. Anyway, I don't think it should pollute the discussion we have here so let's keep it aside.

Reply:
I don't disagree with you.  Just pointing out the matter of resources.